static/app/components/tabs/tab.tsx

```diff
@@ -80,7 +76,6 @@ function BaseTab(
     <TabWrap
       {...tabProps}
       hidden={hidden}
-      disabled={isDisabled}
```

Comment:
`aria-disabled` is provided via `tabProps` and should be sufficient, given that `disabled` is not supported by `li`.